lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -136,15 +136,13 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     const keyMetrics = metricAudits.slice(0, 3);
     const otherMetrics = metricAudits.slice(3);
 
-    const metricsBoxesEl = this.dom.createChildOf(metricAuditsEl, 'div', 'lh-columns');
-    const metricsColumn1El = this.dom.createChildOf(metricsBoxesEl, 'div', 'lh-column');
-    const metricsColumn2El = this.dom.createChildOf(metricsBoxesEl, 'div', 'lh-column');
+    const metricsBoxesEl = this.dom.createChildOf(metricAuditsEl, 'div', 'lh-metrics-container');
 
     keyMetrics.forEach(item => {
```

Comment:
i think can just go with `metricAudits.forEach......` and drop these two lil arrays, right? same order?

Reply:
yeah lol


shouldnt we sort these by weight? the implicit order in the config is not sorted by weight so what we show in the first column is not necessarily the most "key" metrics


![image](https://user-images.githubusercontent.com/4071474/81751168-be15e300-9463-11ea-9fa6-3697c49f9956.png)

Reply:
(no–paul wants the order exactly like this, not ordered by weight)

Reply:
nah i want first col to be FCP, SI, LCP. and second column TTI, TBT, CLS.

the hidden motivation is showing the timing metrics chronological. and the non timing metrics at the end.